README.md

```diff
@@ -1,8 +1,7 @@
-# Conduit Connector Kafka
-![scarf pixel](https://static.scarf.sh/a.png?x-pxid=713ea3ba-66e0-4130-bdd0-d7db4b8706a0)
```

Comment:
I seem to have lost to scarf so I can't corroborate where this one points out or how to create one if necessary (assuming it is) for redpanda. cc @simonl2002